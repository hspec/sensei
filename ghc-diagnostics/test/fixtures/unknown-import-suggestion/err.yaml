version: '1.1'
ghcVersion: ghc-9.10.0
span:
  file: test/fixtures/unknown-import-suggestion/Foo.hs
  start:
    line: 2
    column: 1
  end:
    line: 2
    column: 16
severity: Error
code: 61948
message:
- |-
  Could not find module `Syste.IO'.
  Perhaps you meant System.IO (from base-4.21.0.0)
  Use :set -v to see a list of the files searched for.
hints: []
