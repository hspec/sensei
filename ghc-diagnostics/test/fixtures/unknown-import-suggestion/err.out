test/fixtures/unknown-import-suggestion/Foo.hs:2:1: error: [GHC-61948]
    Could not find module `Syste.IO'.
    Perhaps you meant System.IO (from base-4.21.0.0)
    Use -v to see a list of the files searched for.

